src/Sulu/Bundle/WebsiteBundle/Sitemap/SitemapDumperInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Bundle\WebsiteBundle\Sitemap;
+
+
+/**
+ * Class SitemapDumper
+ */
```

Comment:
Same here, please add better PHPDocs for this interface. You could add the following phpdoc to the methods of concrete implementations to inherit the documentation:

```php
/**
 * {@inheritDoc}
 *
 */
```